src/dmd/cli.d

```diff
@@ -704,6 +704,9 @@ dmd -cov -unittest myprog.d
             "verbose",
             `Enable verbose output for each compiler pass`,
         ),
+        Option("vasm",
```

Comment:
I found [a bug!](https://news.ycombinator.com/item?id=29570425)

Reply:
I'm glad someone reads my sagacious postings on HN!

Reply:
I love your HN posts! 

But to elaborate, when I asked at DConf online what to do about the ever growing list of command line options in DMD, you answered "I got to say 'no' more often". Can you say 'no' to this switch and create a shell script that calls `dmd + obj2asm` instead?

Reply:
The flag could also be just be the compiler running that command for you. 

By far best (e.g. look at what Intel C++ can do) textual emission of assembly on Compiler Explorer is generated from the last IR (i.e. just before actually turning it into actual executable bytes) in the compiler rather than actually dissassembling the binary. dmd on compiler explorer does the latter which is why the output is full of disssssembled padding for example.

Reply:
Doing it in dmd before generating the actual executable bytes would have been a large project that would have been useless for any other purpose, hence would have to be separately maintained. This was a small project, and supporting the complete instruction set is something I regularly do for it anyway, so it is not extra work to keep it up to date.

It's value is also increased as it is designed to be usable for any program that might benefit from a disassembler.